x-pack/plugins/security/public/management/users/edit_user/edit_user_page.tsx

```diff
@@ -205,7 +205,7 @@ export const EditUserPage: FunctionComponent<EditUserPageProps> = ({ username })
                 <FormattedMessage
                   id="xpack.security.management.users.editUserPage.changePasswordDescription"
                   defaultMessage="The user will not be able to log in using their previous
```

Comment:
__optional nit:__ I know it was like that before your PR (not sure why though), but would you mind formatting this default message to have everything on a single line? Feel free to keep it as is as well.